aten/src/ATen/native/cuda/ReflectionPad.cu

```diff
@@ -160,6 +160,215 @@ __global__ void reflection_pad2d_backward_out_kernel(
     gpuAtomicAddNoReturn(&grad_input[index_pair.first], grad_output[index_pair.second]);
   }
 }
+
+template <typename scalar_t>
+__global__ void reflection_pad2d_backward_det_out_kernel(
+    scalar_t* grad_input,
+    const scalar_t* grad_output,
+    int64_t input_dim_x,
+    int64_t input_dim_y,
+    int pad_t,
+    int pad_b,
+    int pad_l,
+    int pad_r,
+    int batch,
+    int channels,
+    int) {
```

Comment:
This parameter is redundant(hence the no name). If I remove it, I will run into compile time errors from the macro expansion of `AT_DISPATCH...`. 

IIRC that's why I left it anyway. I will need to check when I get the opportunity